api/Element.json

```diff
@@ -4573,7 +4573,7 @@
             ],
             "firefox_android": [
               {
-                "version_added": false
+                "version_added": "79"
```

Comment:
A recurring question I've got: a lot (all?) of these features land on Desktop in 59, but in 79 on Android. Is it right that Fennec never enabled Pointer Events by default, but Fenix did?

Reply:
Yeah, the features got enabled at different times. they eventually got enabled for Fenix in version 70: see https://bugzilla.mozilla.org/show_bug.cgi?id=1507495, but Fenix was not publicly released until 79.

Reply:
OK, great. Thank you!